template/php/php-extension.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Skip installing PHP extensions for Core"
```

Comment:
Since you're skipping this should just be removed (as mentioned in the Dockerfile comments).